spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/servlet/UndertowServletWebServerFactoryCustomizer.java

```diff
@@ -36,10 +42,55 @@ public UndertowServletWebServerFactoryCustomizer(ServerProperties serverProperti
 		this.serverProperties = serverProperties;
 	}
 
+	// TODO why was this not being used before now?
```

Comment:
I noticed this Undertow specific customizer exists but was not being called anywhere. Will this be a problem?